docs/contributor/03-60-regions-configuration.md

```diff
@@ -0,0 +1,44 @@
+# Regions and Zones Configuration
+
+Most Kyma plans require specifying a region in the request body as one of the provisioning parameters. The regions' configuration allows you to control which regions are available for the Kyma plan and the platform region. Additionally, every region has a different set of zones.
+## Allowed Regions
+
+The `plansConfiguration` section of the `values.yaml` file contains a list of plans. Every plan has a platform region with allowed regions. To avoid specifying all possible platform regions, define `default` values, for example:
```

Comment:
maybe add a notice here, that plans like free and trial are not supported now, cause their provider depends on the BTP region.